### PR TITLE
Fix golint errors on register package

### PR DESCRIPTION
**Pull request by j-griffith**

### CODE REVIEW

File: pkg/apis/datavolumecontroller/v1alpha1/types.go [hunk 1]
```diff
@@ -31,56 +32,79 @@ type DataVolume struct {
...
 	PVC  *DataVolumeSourcePVC  `json:"pvc,omitempty"`
 }
 
+// DataVolumeSourcePVC provides the parameters to create a Data Volume from an existing PVC
 type DataVolumeSourcePVC struct {
 	Namespace string `json:"namespace,omitempty"`
 	Name      string `json:"name,omitempty"`
 }
 
+// DataVolumeSourceS3 provides the parameters to create a Data Volume from an S3 source
```
Comment (jeffvance) [inline pkg/apis/datavolumecontroller/v1alpha1/types.go]: DataVolumes also work with http sources, AFAIK.

File: pkg/apis/datavolumecontroller/v1alpha1/types.go [hunk 2]
```diff
@@ -31,56 +32,79 @@ type DataVolume struct {
...
 type DataVolumeSourceS3 struct {
-	URL       string `json:"url,omitempty"`
+	//URL is the url of the S3 source
```
Comment (jeffvance) [inline pkg/apis/datavolumecontroller/v1alpha1/types.go]: ditto re http sources

File: pkg/apis/datavolumecontroller/v1alpha1/types.go (hunk 1 above)
Comment (j-griffith) [reply pkg/apis/datavolumecontroller/v1alpha1/types.go]: ?  This comment is for a DataVolumeS3 struct?

File: pkg/apis/datavolumecontroller/v1alpha1/types.go (hunk 2 above)
Comment (j-griffith) [reply pkg/apis/datavolumecontroller/v1alpha1/types.go]: Same as above, this is the DataVolumeSourceS3.URL member.

File: pkg/apis/datavolumecontroller/v1alpha1/types.go (hunk 1 above)
Comment (jeffvance) [reply pkg/apis/datavolumecontroller/v1alpha1/types.go]: Nevermind... I didn't notice that.

**Comment by jeffvance** [comment]
lgtm

**Pull request merged**